lib/ansible/modules/network/fortimanager/fmgr_fwobj_address.py

```diff
@@ -0,0 +1,887 @@
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    "metadata_version": "1.1",
+    "status": ["preview"],
+    "supported_by": "community"
+}
+
+DOCUMENTATION = '''
+---
+module: fmgr_fwobj_address
+version_added: "2.8"
+author:
+    - Luke Weighall (@lweighall)
+    - Andrew Welsh (@Ghilli3)
+    - Jim Huber (@p4r4n0y1ng)
+short_description: Allows the management of firewall objects in FortiManager
+description:
+  -  Allows for the management of ipv4, ipv6, and multicast address objects within FortiManager
+
+options:
+  host:
+    description:
+      - The FortiManager Address.
+    required: true
+  username:
+    description:
+      - The username used to authenticate with the FortiManager.
+    required: true
+  password:
+    description:
+      - The password associated with the username account.
+    required: true
+
+  adom:
+    description:
+      - The ADOM the configuration should belong to.
+
+  allow_routing:
+    description:
+      - Enable/disable use of this address in the static route configuration.
+    choices: ['enable', 'disable']
+
+  associated_interface:
+    description:
+      - Associated interface name.
+
+  cache_ttl:
+    description:
+      - Minimal TTL of individual IP addresses in FQDN cache. Only applies when type = wildcard-fqdn
+
+  color:
+    description:
+      - Color of the object in FortiManager GUI
+      - Takes integers 1-32
+
+  comment:
+    description:
+      - Comment for the object in FortiManager
+
+  country:
+    description:
+      - Country name. Required if type = geographic
+
+  end_ip:
+    description:
+      - End IP. Only used when ipv4 = range.
+
+  group_members:
+    description:
+      - Address group member. If this is defined w/out group_name, the operation will fail.
+
+  group_name:
+    description:
+      - Address group name. If this is defined in playbook task, all other options are ignored.
+
+  ipv4:
+    description:
+      - Type of IPv4 Object.
+      - Must not be specified with either multicast or ipv6 parameters
+    choices: ['ipmask', 'iprange', 'fqdn', 'wildcard', 'geography', 'wildcard-fqdn', 'group']
+
+  ipv4addr:
+    description:
+      - IP and network mask. If only defining one IP use this parameter. (i.e. 10.7.220.30/255.255.255.255)
+      - Can also define subnets (i.e. 10.7.220.0/255.255.255.0)
+      - Also accepts CIDR (i.e. 10.7.220.0/24)
+      - If Netmask is omitted after IP address, /32 is assumed.
+      - When multicast is set to Broadcast Subnet the ipv4addr parameter is used to specify the subnet
+
+  ipv6:
+    description:
+      - Puts module into IPv6 mode.
+      - Must not be specified with either ipv4 or multicast parameters
+    choices: ['ip', 'iprange', 'group']
+
+  ipv6addr:
+    description:
+      - IPv6 Address in Full
```

Comment:
```suggestion
      - IPv6 address in full.
```

Reply:
it might be worth adding an example of what you mean by IPv6 address in full, like you have in the example - 2001:0db8:85a3:0000:0000:8a2e:0370:7334